src/group_theory/commutator.lean

```diff
@@ -204,3 +204,13 @@ begin
 end
 
 end subgroup
+
+variables (G)
+
+def commutators : set G :=
+{g | ∃ g₁ g₂ : G, ⁅g₁, g₂⁆ = g}
+
+lemma commutators_def : commutators G = {g | ∃ g₁ g₂ : G, ⁅g₁, g₂⁆ = g} := rfl
+
```

Comment:
I think the lemmas `mem_commutators_iff` and `commutator_mem_commutators` would be useful to have.